applications/adv_networking_bench/adv_networking_bench_default_rx.yaml

```diff
@@ -48,28 +48,15 @@ advanced_network:
         - local
       num_bufs: 30720
       buf_size: 1064
-    - name: "Default_RX_CPU"
```

Comment:
needed by `Data` queue for HDS

Reply:
Actually, no, this is not needed. Data_RX_CPU is used for HDS

Reply:
my bad